common/app/model/content.scala

```diff
@@ -233,7 +237,52 @@ object Content {
     }
   }
 
-  def make(apiContent: contentapi.Content): Content = {
+  object BodyBlocks {
+    def make(blocks: Option[Blocks]): Seq[BodyBlocks] =
+    blocks.toSeq.flatMap { blocks =>
```

Comment:
ok @philwills  I've moved it into its own file and made all the variable names sensible, looks better now :grinning: